addons/project/views/rating_views.xml

```diff
@@ -6,14 +6,31 @@
         <field name="inherit_id" ref="rating.rating_rating_view_tree"/>
         <field name="mode">primary</field>
         <field name="arch" type="xml">
-            <field name="res_name" position="attributes">
-                <attribute name="string">Task</attribute>
+            <field name="rating_text" position="before">
+                <field name="create_date" position="move"/>
+                <field name="rated_partner_id" position="move"/>
+                <field name="partner_id" position="move"/>
+                <field name="parent_res_name" position="move"/>
+                <field name="res_name" position="move"/>
+                <field name="feedback" position="move"/>
+            </field>
```

Comment:
So, I think we could simply put `rating_text` before the `create_date` to have the same result.
What do you think ? 🤔 
```suggestion
            <field name="create_date" position="before">
                <field name="rating_text" position="move"/>
            </field>
```

Reply:
@xavierbol Already fixed [here](https://github.com/odoo/odoo/commit/cfd5837a24e41fd97b58a18bb94f3f1136f05326#diff-37ca32b0e15cca6c78c4c3922781b6d75da7bcbf95a51bfa810d794560110169R10)